### PR TITLE
Adds Chinese translations to README

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

Adds Chinese translation to README for our Chinese community members
翻译了README文档。我没有在中文语言环境开发的经验，欢迎与我用中文沟通以及帮我差错

## Test Plan

N/A

Although detailed reviews from other experienced Chinese developers would be nice.
欢迎有在中文环境下开发的朋友们为我差错，纠正用词。

## Related PRs and Issues

#2584

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+如需完整的 API 界面文档，请访问 [https://appwrite.io/docs](https://appwrite.io/docs)。如需更多教程、新闻和公告，请订阅我们的 [博客](https://medium.com/appwrite-io) 和 加入我们的[Discord 社区](https://discord.gg/GSeTUeA)。
+
+### 开发套件
+
+以下是当前支持的平台和语言列表。如果您想帮助我们为您选择的平台添加支持，您可以访问我们的 [SDK 生成器](https://github.com/appwrite/sdk-generator) 项目并查看我们的 [贡献指南]( https://github.com/appwrite/sdk-generator/blob/master/CONTRIBUTING.md）。
+
+#### 客户端
+* ✅  &nbsp; [Web](https://github.com/appwrite/sdk-for-web) (由 Appwrite 团队维护)
+* ✅  &nbsp; [Flutter](https://github.com/appwrite/sdk-for-flutter) (由 Appwrite 团队维护)
+* ✅  &nbsp; [Apple](https://github.com/appwrite/sdk-for-apple) - **公侧** (由 Appwrite 团队维护)
```
Comment (adamyy) [inline README-CN.md:124]: ~公测~ 公测

File: README-CN.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+找不到需要的的 SDK？ - 欢迎通过发起PR来帮助我们完善Appwrite的软件生态环境 [SDK 生成器](https://github.com/appwrite/sdk-generator)!
+
+
+## 软件架构
+
+![Appwrite 软件架构](docs/specs/overview.drawio.svg)
+
+Appwrite 使用高拓展性的微服务架构。此外，Appwrite 支持多种 API（REST、WebSocket 和 即将推出的 GraphQL），来迎合您的个性化开发习惯。
+
+Appwrite API 界面层利用后台缓存和任务委派来提供极地的反应时间。后台的 Worker 代理还允许您使用消息队列来处理负载，并精确控制硬件合理分配和成本。您可以在 [贡献指南](CONTRIBUTING.md#architecture-1) 中了解有关我们架构的更多信息。
```
Comment (adamyy) [inline README-CN.md:151]: ~极地的反应时间~ 极速的响应时间

**Review by adamyy — COMMENTED** [review]
Several suggestions regarding wording

File: README-CN.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+<!-- [![Hacktoberfest](https://img.shields.io/static/v1?label=hacktoberfest&message=friendly&color=90a88b&style=flat-square)](https://hacktoberfest.appwrite.io) -->
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord&style=flat-square)](https://appwrite.io/discord?r=Github)
+[![Docker Pulls](https://img.shields.io/docker/pulls/appwrite/appwrite?color=f02e65&style=flat-square)](https://hub.docker.com/r/appwrite/appwrite)
+[![Build Status](https://img.shields.io/travis/com/appwrite/appwrite?style=flat-square)](https://travis-ci.com/appwrite/appwrite)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+[![翻译](https://img.shields.io/badge/translate-f02e65?style=flat-square)](docs/tutorials/add-translations.md)
+[![周边商店](https://img.shields.io/badge/swag%20store-f02e65?style=flat-square)](https://store.appwrite.io)
+
+Appwrite 是一个可应用于网页，移动，和后端开发的端到端后端平台, 以一系列 Docker 容器化的微服务形式包装发行。Appwrite 以高度抽象化的界面简化了从零编写 API 的繁琐过程，在保证软件安全的前提下给开发者一个高效的开发过程。
```
Comment (yuqil725) [inline README-CN.md:19]: Appwrite是一个基于dcoker的端到端开发者平台，其容器化的微服务库可应用于网页端，移动端，以及后端。Appwrite 通过视觉化界面极简了从零编写 API 的繁琐过程，在保证软件安全的前提下为开发者创造了一个高效的开发环境。

File: README-CN.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+```powershell
+docker run -it --rm ,
+    --volume /var/run/docker.sock:/var/run/docker.sock ,
+    --volume ${pwd}/appwrite:/usr/src/code/appwrite:rw ,
+    --entrypoint="install" ,
+    appwrite/appwrite:0.12.1
+```
+
+运行后，可以在浏览器上访问 http://localhost 找到 Appwrite 控制台。请注意，在非 Linux 本机主机上，安装完成后服务器可能需要几分钟才能启动。
```
Comment (yuqil725) [inline README-CN.md:85]: 在非 Linux 的本机主机上完成安装后，服务器可能需要几分钟才能启动。

File: README-CN.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+## 贡献代码
+
+所有代码贡献 - 包括来自具有直接提交更改权限的贡献者 - 都必须提交PR请求并在合并分支之前得到核心开发人员的批准。这是为了确保正确审查所有代码。
+
+我们欢迎所有人提交PR！如果您愿意提供帮助，可以在 [贡献指南](CONTRIBUTING.md) 中了解有关如何为项目做出贡献的更多信息。
+
+## 安全
+
+对于安全问题，请发送电子邮件至 [security@appwrite.io](mailto:security@appwrite.io)，而不是在 GitHub 上发布公开问题。
```
Comment (yuqil725) [inline README-CN.md:161]: 为了保护您的隐私，请避免在GitHub 上发布安全问题。发送问题至 [security@appwrite.io](mailto:security@appwrite.io)，我们将为您做更细致的解答。

**Comment by eldadfux** [comment]
@gewenyu99 waiting for your sign, let me know when we can approve this one. Would be nice to add links for the English and Chinese versions below the logo.

**Comment by gewenyu99** [comment]
@eldadfux I think this should be ready to merge. I added links to translations with formatting suggestions from Christy and Bradley.

We're now duo-lingo :D

**Comment by eldadfux** [comment]
Awesome! all changes merged!

**Pull request merged**